src/test/java/uk/gov/pay/adminusers/resources/InviteResourceServiceCompleteIT.java

```diff
@@ -30,8 +30,12 @@ void shouldReturn200withDisabledInviteLinkingToCreatedUser_WhenPassedAValidInvit
                 .withEmail(email)
                 .withPassword(password)
                 .insertServiceInvite();
+
+        Map<String, String> payload = Map.of("second_factor", "SMS");
```

Comment:
Possibly it's better if all the tests check setting the `second_factor` method to "APP" as it is less likely to be a default if we were setting a default anywhere?

Reply:
Agreed.  I thought about doing this in the first place but then didn't.